### DISCUSSION TITLE
support for external storage

### DISCUSSION THREAD

**Discussion by bentouspam** [music-assistant/support#550] [Feature requests and ideas]
Hello,

I would like to be able to play the music stored on another device, such as a CIFS or a NFS share, or a DLNA server.
with HA OS you cannot have access to the shell or you may not be able to handle a fstab file...
copying all the music on my small raspberry is not the solution either ;-)

another thing is the volume level.
it should be on a logarithmic scale. on the low volume moving from 5% to 10% is quite tricky to adjust if your device is a bit powerpull.

music-assistant is very nice. shall we espect a movie-assistant ??
I have a lot of movies and my hundreds of CD are ripped on my synology, I cannot use music-assistant, even with my free spotify account... :-(

**Comment by OzGav** [comment]
Have a look here for help with this https://github.com/music-assistant/hass-music-assistant/discussions/452

**Comment by RickeyWard** [comment]
manually mounting is a solution, but just letting you add UNC paths would ne nice. I realize there's a lot of work behind that, but its still a nice to have.

**Comment by marcelveldt** [comment]
There is an existing feature request for this so maybe better but your votes there ?
